core/domain/user_services.py

```diff
@@ -405,3 +407,120 @@ def get_email_preferences(user_id):
             if email_preferences_model is None
             else email_preferences_model.site_updates)
     }
+
+
+class UserContributions(object):
+    """Value object representing a user's contributions."""
+    def __init__(
+            self, user_id, created_exploration_ids, edited_exploration_ids):
+        self.user_id = user_id
+        self.created_exploration_ids = created_exploration_ids
+        self.edited_exploration_ids = edited_exploration_ids
+
+    def validate(self):
+        if not isinstance(self.user_id, basestring):
+            raise utils.ValidationError(
+                'Expected user_id to be a string, received %s' % self.user_id)
+        if not isinstance(self.created_exploration_ids, list):
+            raise utils.ValidationError(
+                'Expected created_exploration_ids to be a list, received %s' 
+            % self.created_exploration_ids)
+        if not isinstance(self.edited_exploration_ids, list):
+            raise utils.ValidationError(
+                'Expected edited_exploration_ids to be a list, received %s' 
+            % self.edited_exploration_ids)
+        if not self.user_id:
+            raise utils.ValidationError('No user id specified.')
+        for exploration_id in self.created_exploration_ids:
+            if not isinstance(exploration_id, basestring):
+                raise utils.ValidationError(
+                    '''Expected exploration_id in created_exploration_ids 
```

Comment:
raise utils.ValidationError(
                    'Expected exploration_id in created_exploration_ids '
                    'to be a string, received %s' % exploration_id)

(We generally only use triple quotes for docstrings. Note that, in Python, you can split a single-quote string across multiple lines like the above -- this is as opposed to JS, where you need '+' signs at the end.)